hermes/bin/src/runtime_extensions/hermes/mod.rs

```diff
@@ -13,47 +11,3 @@ pub(crate) mod json;
 pub(crate) mod kv_store;
 pub(crate) mod localtime;
 pub(crate) mod logging;
-
-/// Hermes extensions state
-pub(crate) struct State {
```

Comment:
It is important to let the runtime extensions know of a new context before they receive it.
See requested changes PR for details